ios/scripts/asc_download_profile.py

```
@@ -0,0 +1,151 @@
+        raise RuntimeError("set ASC_API_KEY_PATH or ASC_API_KEY_P8_BASE64")
+    return (signing_input + b"." + _b64u(signature)).decode()
+
+
+def _asc_error_code(body: Dict) -> str:
+    try:
+        return str(((body.get("errors") or [{}])[0]).get("code", "unknown"))
+    except Exception:
+        return "unknown"
+
+
+def _request(token: str, method: str, path: str):
```

Ruff (0.15.21)

[warning] Missing return type annotation for private function `_request`

(ANN202)